environment.yml

```diff
@@ -18,8 +18,8 @@ dependencies:
   - psutil>=5.1.0
   - sphinx
   - sphinx_rtd_theme
-  - dask
-  - distributed>=1.21.8
+  - dask==0.18.1
+  - distributed==1.22.0
```

Comment:
does it break with >= ?

Reply:
We want to be able to reproduce the behaviour of `helm install stable/dask`, and that, as you might see [here](https://github.com/helm/charts/tree/master/stable/dask) , installs these specific versions. The Docker image doesn't seem to start the bokeh server on later versions of dask, and this version of dask is locked into this version of distributed AFAIU.

Reply:
OK